gh-110383: Fix documentation profile cumtime fix (#112221)
python/cpython: Doc/library/profile.rst

        1    0.000    0.000    0.000    0.000 _compiler.py:598(_code)
        1    0.000    0.000    0.000    0.000 _parser.py:435(_parse_sub)

> The first line indicates that 214 calls were monitored. Of those calls, 207

Reviewer: Let's not change unrelated whitespace.
```suggestion
The first line indicates that 214 calls were monitored.  Of those calls, 207
```

Author: Sure, I also thought it was unrelated, just pretty close to context too. Returned

Reviewer: Thanks. I also prefer one space, but we allow either one or two: https://devguide.python.org/documentation/style-guide/#use-of-whitespace

were :dfn:`primitive`, meaning that the call was not induced via recursion. The

> next line: ``Ordered by: cumulative time`` indicates rows sorting order
> for the table below. The column headings include:

Reviewer: The table is above, not below. Maybe reword like this?

```suggestion
next line: ``Ordered by: cumulative time`` indicates the output is sorted
by the ``cumtime`` values. The column headings include:
```
